client/components/sub-masterbar-nav/README.md

```diff
@@ -0,0 +1,37 @@
+Sub Masterbar Nav
+=================
+
+This component displays a navigation header under the masterbar.
+
+### How to use:
+
+The component should be inserted into a ```Main``` component.  
```

Comment:
Not too relevant, but do we really need triple backticks everywhere? Aren't single ones enough? :smile: